packages/frontend/editor-ui/src/features/insights/components/charts/insightChartProps.ts

```diff
@@ -0,0 +1,7 @@
+import type { InsightsByTime, InsightsSummaryType, InsightsDateRange } from '@n8n/api-types';
+
+export type ChartProps = {
```

Comment:
Nice! I hate that we have a lot of repetition in the chart components.
Maybe we can improve it further by creating a composable for the common parts, if possible. If not, then this is still an improvement!